### PR TITLE
remove unhelpful logging

### PR DESCRIPTION
**Pull request by michaelkirk**
This is open for line item veto

PTAL @charlesmchen

### CODE REVIEW

File: src/Contacts/PhoneNumber.m [hunk 1]
```diff
@@ -21,11 +21,6 @@ + (PhoneNumber *)phoneNumberFromText:(NSString *)text andRegion:(NSString *)regi
     NSError *parseError   = nil;
     NBPhoneNumber *number = [phoneUtil parse:text defaultRegion:regionCode error:&parseError];
 
-    if (parseError) {
-        DDLogDebug(@"Issue while parsing number: %@", [parseError description]);
```
Comment (charlesmchen) [inline src/Contacts/PhoneNumber.m]: Don't we still want to early exit in this case?

Comment (michaelkirk) [reply src/Contacts/PhoneNumber.m]: definitely yes. Good catch.

**Comment by michaelkirk** [comment]
PTAL @charlesmchen

**Review by charlesmchen — APPROVED** [review]
LGTM

**Pull request merged**